pkg/webreg/webreg.go

```diff
@@ -734,15 +734,78 @@ syntax highlight all commands as bash.
 </p>
 
 <p>
-Sharing files between steps is supported via a shared directory. All
-containers will have an environment variable <code>SHARED_DIR</code> which
-contains the path of the shared directory. An artifacts directory also exists
-for steps that produce artifacts. The path for the artifacts directory is
-stored in the <code>ARTIFACTS_DIR</code> environment variable.
+A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
 </p>
 
+<h4 id="execution"><a href="#execution">Step Execution Environment</a></h4>
 <p>
-A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
+While a step simply defines a set of commands to run in a container image,
+by virtue of executing within a <code>ci-operator</code> workflow, the commands
+have a number of speical considerations for their execution environment.
+The commands can expect a set of environment variables to exist that inform
+them of the context in which they run; commands in steps can communicate to
+other steps via a shared directory in their filesystem.
```

Comment:
It would be good to mention the "upstream" environment that comes from `ci-operator`/Prow (I don't know if we have something in our documentation already that can be linked here).

Reply:
Yeah, I'll link those but generally when people are doing something with those vars they're doing something crazy and should put down the keyboard ;)